src/elements/IncompleteFormAlert/IncompleteFormAlert.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {
+	Text,
+	View,
+	Image,
+} from 'react-native';
+import useGlobal from '@state';
+import {
+	TextButton,
+	Modal,
+} from '@elements';
+import { Alert } from '@state/index.types';
+import { useScheme } from '@util/colorSchemes';
+import typography from '@util/typography';
+import styles from './IncompleteFormAlert.styles';
+
+export default () => {
+	const [ globalState, globalActions ] = useGlobal() as any;
+	const { alert }: { alert: Alert } = globalState;
+	const { clearAlert } = globalActions;
+
+	const scheme = useScheme();
+
+	const handleCloseButtonPress = () => {
+		clearAlert();
+	};
+
+	const handleDismiss = () => {
+		if (alert.dismissable) {
+			clearAlert();
+		}
+	};
+
+	return (
+		<Modal
+			style={styles.container}
+			title={alert?.title || 'Alert'}
+			palette="secondary"
+			open={alert !== undefined}
+			onDismiss={handleDismiss}
+		>
+
+			<View style={styles.body}>
+
+				<View style={styles.imageWrapper}>
+					<Image
+						style={styles.imageContainer}
+						source={require('@assets/images/Ellipse.png')}
+					/>
+					<Text style={typography.h4}>BANANA</Text>
+				</View>
+
+				<View style={styles.textContainer}>
+					<Text style={typography.body1}>
+						{alert?.message || 'Uh oh, an unknown error occurred!'}
+					</Text>
+				</View>
+
+				<View style={styles.buttonWrapper}>
+					<View style={styles.leftButton}>
+						<TextButton
+							text="NO, KEEP IT"
+							style={{
+								width: 120,
+							}}
+							buttonStyle={{
+								default: scheme.tertiary,
+								pressed: scheme.secondary,
+								disabled: scheme.disabled,
+							}}
+							onPress={handleCloseButtonPress}
+						/>
+					</View>
+					<View style={styles.rightButton}>
+						<TextButton
+							text="YES, CANCEL"
+							style={{
+								width: 120,
+							}}
+							buttonStyle={{
+								default: scheme.primary,
+								pressed: scheme.secondary,
+								disabled: scheme.disabled,
+							}}
+							onPress={handleCloseButtonPress}
+						/>
```

Comment:
The `YES, CANCEL` button and the `NO, KEEP IT` buttons both trigger the same function. Would these buttons have different behaviors? For instance, `NO KEEP IT` would do nothing except dismiss, and `YES CANCEL` would exit the form?

Reply:
Thank you Edgar for taking your time to review this. 
Yes, you are right, the "YES, CANCEL" button will exit the form. The thing is this modal will be used in many different forms and I did not know how to write that generic function I mean where to take our user. Please let me know if you can help me with this.

Reply:
I'd be happy help!